.teamcity/API/buildTypes/DeployAPI.kt

```diff
@@ -18,58 +19,30 @@ object DeployAPIBuild : BuildType ({
         // Default setting is to clean before next build
         swabra {
         }
+
+        nuGetFeedCredentials {
+            feedUrl = "%azureArtifacts.feed.nuget%"
+            username = "%azureArtifacts.edFiBuildAgent.userName%"
+            password = "%azureArtifacts.edFiBuildAgent.accessToken%"
+        }
     }
 
     dependencies {
-        artifacts(BranchAPIBuild) {
+        artifacts(PullRequestAPIBuild) {
```

Comment:
This seems odd to me - publishing packages from a pull request instead of from the `main` build.

Reply:
I see the main build is Branch*Build for the vcs restriction to default now. Sorry for the confusion on my part. I will fix.